web-common/src/components/dialog/tabs/__stories__/DialogTabs.stories.svelte

```diff
@@ -0,0 +1,86 @@
+<script lang="ts">
+  import {
+    Dialog,
+    DialogOverlay,
+    DialogTitle,
+  } from "@rgossiaux/svelte-headlessui";
+  import { Meta, Story } from "@storybook/addon-svelte-csf";
+  import * as DialogTabs from "../";
+  import Button from "../../../button/Button.svelte";
+
+  let selectedTabIndex = 0;
+  const tabs = ["Tab 1", "Tab 2", "Tab 3"];
+
+  function handleNextTab() {
```

Comment:
Should we wrap this next/back/done functionality in a component as well?

Reply:
Oh interesting idea – maybe if/when we have another multi-panel dialog we can look at doing that.